services/dualmotors.md

```diff
@@ -0,0 +1,34 @@
+# Dual Motors
+
+    identifier: 0x1529d537
+    group: motor
+
+A synchronized pair of motors.
+
+## Registers
+
+    rw speed @ value {
+        left: i1.15 /
+        right: i1.15 /
+    }
+
+Relative speed of the motors. Use positive/negative values to run the motor forwards and backwards.
+Positive is recommended to be clockwise rotation and negative counterclockwise. A speed of ``0`` 
```

Comment:
I think you want to remove "clockwise" - typically you would want (1,1) to go forward but if both wheels run clockwise you will spin.

Reply:
removed